graphql-upload/services/local/schema.js

```diff
@@ -0,0 +1,54 @@
+const { makeExecutableSchema } = require("@graphql-tools/schema");
+
+// does not work
+// const { GraphQLUpload } = require("graphql-upload");
+
+// does work
+const { GraphQLUpload } =require("@graphql-tools/links");
```

Comment:
@yaacovCR this is the issue I was having in this thread https://github.com/ardatan/graphql-tools/issues/671#issuecomment-817182683. The original `GraphQLUpload` throws an error.